en/docs/quick-start-guides/deploy-your-first-integration.md

```diff
@@ -0,0 +1,198 @@
+# Deploy Your First Integration
```

Comment:
Should the name be `Deploy your first Integration` or `Deploy your first Integration as API` to align with the type

Reply:
Decided offline to go with `Deploy your first Integration as API`